nautobot/dcim/models/device_components.py

```diff
@@ -598,59 +708,41 @@ def to_csv(self):
             self.mgmt_only,
             self.description,
             self.get_mode_display(),
+            self.parent_interface.name if self.parent_interface else None,
+            self.bridge.name if self.bridge else None,
         )
 
     def clean(self):
         super().clean()
 
-        # Virtual interfaces cannot be connected
-        if self.type in NONCONNECTABLE_IFACE_TYPES and (self.cable or getattr(self, "circuit_termination", False)):
-            raise ValidationError(
-                {
-                    "type": "Virtual and wireless interfaces cannot be connected to another interface or circuit. "
-                    "Disconnect the interface or choose a suitable type."
-                }
-            )
+        # LAG validation
+        if self.lag is not None:
 
-        # An interface's LAG must belong to the same device or virtual chassis
-        if self.lag and self.lag.device != self.device:
-            if self.device.virtual_chassis is None:
-                raise ValidationError(
-                    {
-                        "lag": f"The selected LAG interface ({self.lag}) belongs to a different device ({self.lag.device})."
-                    }
-                )
-            elif self.lag.device.virtual_chassis != self.device.virtual_chassis:
-                raise ValidationError(
-                    {
-                        "lag": f"The selected LAG interface ({self.lag}) belongs to {self.lag.device}, which is not part "
-                        f"of virtual chassis {self.device.virtual_chassis}."
-                    }
-                )
-
-        # A virtual interface cannot have a parent LAG
-        if self.type == InterfaceTypeChoices.TYPE_VIRTUAL and self.lag is not None:
-            raise ValidationError({"lag": "Virtual interfaces cannot have a parent LAG interface."})
-
-        # A LAG interface cannot be its own parent
-        if self.present_in_database and self.lag_id == self.pk:
-            raise ValidationError({"lag": "A LAG interface cannot be its own parent."})
+            # A LAG interface cannot be its own parent
+            if self.lag_id == self.pk:
+                raise ValidationError({"lag": "A LAG interface cannot be its own parent."})
```

Comment:
Is the `present_in_database` check no longer needed? If so, why?

Reply:
This check is just ensuring `self` and `self.lag` aren't the same object which doesn't require either to be in the DB. Since `pk` is generated on init with UUID, we don't have the same `None == None` check with Integer based IDs.